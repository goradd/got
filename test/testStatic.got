package testOut

import (
	"bytes"
	"fmt"
)

type Translater interface {
	Translate(string, *bytes.Buffer)
}

func TestStatic(buf *bytes.Buffer) {
{{ Here {{go buf.WriteString("is") }} some code wrapping text escaping to go. }}

{{
<p>
{{! Escaped html < }}
</p>
}}

{{h
	This is text
	that is both escaped and has
	html paragraphs and breaks inserted.
}}

}

func translateTest(t Translater, buf *bytes.Buffer) {

{{t Translate me to some language }}

{{!t Translate & escape me > }}

}