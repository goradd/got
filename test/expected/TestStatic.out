Here is some code wrapping text escaping to go.
Testing quotes: "'`
<p>
Escaped html &lt;
</p>
<p>This is text<br>
	that is both escaped and has<br>
	html paragraphs and breaks inserted.<br>
</p>

This should be testStatic: testStatic
